docs/architecture/dapr-for-net-developers/dapr-at-20000-feet.md

```diff
@@ -0,0 +1,257 @@
+
+Dapr also runs in [containerized environments](https://docs.dapr.io/concepts/overview/#kubernetes-hosted), such as Kubernetes. Figure 2-6 shows Dapr running in a separate side-car container along with the application container in the same Kubernetes pod.
+
+![Sidecar architecture](./media/kubernetes-hosted-dapr-sidecar.png)
+
+**Figure 2-6**. Kubernetes-hosted Dapr sidecar
+
+## Dapr performance considerations
+
+As you've seen, Dapr exposes a sidecar architecture to decouple your application from distributed application capabilities. Invoking a Dapr operation requires at least one out-of-process network call. Figure 2-7 presents an example of a Dapr traffic pattern.
+
+![Dapr traffic patterns](./media/dapr-traffic-patterns.png)
+
+**Figure 2-7**. Dapr traffic patterns
+
+Looking at the previous figure, one might question the latency and overhead incurred for each call.  
+
+The Dapr team has invested heavily in performance. A tremendous amount of engineering effort has gone into making Dapr efficient. Calls between Dapr sidecars are always made with gRPC, which delivers high performance and small, binary payloads. In most cases, the additional overhead should be less than 1 millisecond.
+
+To increase performance, developers can call the Dapr building blocks with gRPC.
+
+gRPC is a modern, high-performance framework that evolves the age-old [remote procedure call (RPC)](https://en.wikipedia.org/wiki/Remote_procedure_call) protocol. gRPC uses HTTP/2 for its transport protocol, which provides significant performance enhancements over HTTP RESTFul service, including:
+
+- Multiplexing support for sending multiple parallel requests over the same connection - HTTP 1.1 limits processing to one request/response message at a time.
+- Bidirectional full-duplex communication for sending both client requests and server responses simultaneously.
+- Built-in streaming enabling requests and responses to asynchronously stream large data sets.
+
+## Dapr and service meshes
+
+Service mesh is another rapidly evolving technology for distributed applications.
+
+A service mesh is a configurable infrastructure layer with built-in capabilities to handle service-to-service communication, resiliency, load balancing, and telemetry capture. It moves the responsibility for these concerns out of the services and into the service mesh layer. Like Dapr, a service mesh also follows a sidecar architecture.
+
+Figure 2-8 shows an application that implements service mesh technology.
+
+![Service Mesh](./media/service-mesh-with-side-car.png)
+
+**Figure 2-8**. Service mesh with a side car.
+
+Note in the previous figure how messages are intercepted by a proxy that runs alongside each service. Each proxy can be configured with traffic rules specific to the service. It understands messages and can route them across your services and the outside world.
```

Comment:
```suggestion
The previous figure depicts how messages are intercepted by a proxy that runs alongside each service. Each proxy can be configured with traffic rules specific to the service. It understands messages and can route them across your services and the outside world.
```

Either use a note alert, or remove "Note". Try to avoid starting a sentence with "Note".